src/AccessibilityInsights.Extensions.GitHub/NoFailuresIssueFormatter.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+
+namespace AccessibilityInsights.Extensions.GitHub
+{
+    public class NoFailuresIssueFormatter : IIssueFormatter
+    {
+        public IssueInformation IssueInfo { get; }
+        public NoFailuresIssueFormatter(IssueInformation issueInfo)
+        {
+            this.IssueInfo = issueInfo;
+        }
+        public string GetFormattedBody()
+        {
+            return string.Format(
+                "The following accessibility issue that needs investigation.\n\n" +
+                "**App:** {0}\n\n" +
+                "**Element path:** {1}\n\n" +
+                "**Issue Details:** []\n\n" +
+                "**How To Fix:** []\n\n",
+                IssueFormatterFactory.GetStringValue(this.IssueInfo.ProcessName),
```

Comment:
how about putting it in a resource string?

Reply:
I don't think we need it here, can't think of a case where we're going to need to localize it or something